src/Tools/Widgets.twee

```diff
@@ -11,22 +11,23 @@
 			<th>Goods</th>
 			<th>Cost</th>
 		</tr>
-		<<for _key, _item range _target>>
+		<<for _i to 0; _i lt _target.length; _i++>>
```

Comment:
@ryceg Is there a neater way of iterating here? Maybe `_item in range _target` or something?

Reply:
SugarCube has conditional, 3 part, and range `<<for>>` loops; https://www.motoslave.net/sugarcube/2/docs/#macros-macro-for